com.unity.render-pipelines.core/Editor/Volume/VolumeComponentEditor.cs

```diff
@@ -75,6 +75,25 @@ public VolumeComponentEditorAttribute(Type componentType)
     /// <seealso cref="VolumeComponentEditorAttribute"/>
     public class VolumeComponentEditor
     {
+        class Texts
+        {
+            public static GUIContent overrideSettingText { get; } = EditorGUIUtility.TrTextContent("", "Override this setting for this volume.");
+            public static GUIContent allText { get; } = EditorGUIUtility.TrTextContent("ALL", "Toggle all overrides on. To maximize performances you should only toggle overrides that you actually need.");
+            public static GUIContent noneText { get; } = EditorGUIUtility.TrTextContent("NONE", "Toggle all overrides off.");
+
+            public static string toggleAllText { get; } = L10n.Tr("Toggle All");
+        }
+
+        Vector2? m_OverrideToggleSize;
```

Comment:
I am not familiar with that notation. what does the ? mean after the type

Reply:
This is making a nullable value type (since c# 8 you can do it on reference types).

Basically it allow us to set null to the variable(default value)

I am using it here to allow me to check if it has been initialized.

For instance imagine a `bool` that is binded to UI. bool will only have true(yes) or false(no) as possibilities, but you want to make sure that the user selects if the option is wanted or not. So, you could not default to `false`, So instead of having another bool to check if the the user has selected yes or no for this, you make a `bool?`
now our `bool?` has three states null, true, false. This will allow us to check easily if the user has selected a value for it.

So, to obtain the value of a nullable reference or value type you could do it by `.HasValue` (but it think performance wise will be the same as doing `==null`), and to obtain the value stored in our case a `Vector2` simply use .Value.

Reply:
thanks for the tip!